pkg/apis/kubermatic/v1/cluster.go

```diff
@@ -292,6 +292,8 @@ const (
 
 	ClusterConditionEtcdClusterInitialized ClusterConditionType = "EtcdClusterInitialized"
 
+	ClusterConditionUpdateProgress ClusterConditionType = "UpdateProgress"
```

Comment:
Because the status can be "no update currently in progress", so the condition should not be called "UpdateInProgress". Maybe "UpdateStatus" is clearer?

Reply:
In that case, UpgradeProgress is fine.